components/org.wso2.carbon.identity.idp.metadata.saml2/src/main/java/org/wso2/carbon/identity/idp/metadata/saml2/builder/IDPMetadataBuilder.java

```diff
@@ -32,17 +32,19 @@
 import org.wso2.carbon.identity.idp.metadata.saml2.IDPMetadataConstant;
 import org.wso2.carbon.identity.idp.metadata.saml2.util.BuilderUtil;
 import org.wso2.carbon.idp.mgt.MetadataException;
+import org.opensaml.saml2.metadata.AttributeProfile;
```

Comment:
We are using open-saml version 3 in this branch. Hence, can you please use the dependencies from opensaml 3 since the travis build is failing because of these imports?